tap_hubspot_beta/client_base.py

```diff
@@ -309,6 +309,10 @@ def schema(self):
             if not field.get("deleted"):
                 property = th.Property(field_name, self.extract_type(field))
                 properties.append(property)
+            
+            if field.get("type") == "date" and hasattr(self, "date_fields"):
+                self.date_fields.append(field_name)
```

Comment:
Instead of adding to this new `self.date_fields` you added can we just reuse `self.datetime_fields`?